pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design.md

```diff
@@ -184,5 +184,5 @@ Here are some examples that we hope inspire you to create responsive AMP pages:
 #### Made by AMP
 
 - [Examples]({{g.doc('/content/amp-dev/documentation/examples/index.html', locale=doc.locale).url.path}})
-- [AMP Start templates](https://www.ampstart.com/)
+- [AMP Start templates]({{g.doc('/content/amp-dev/documentation/templates/index.html', locale=doc.locale).url.path}})
```

Comment:
Rephrase link text to just "Templates" - also in the i18n versions.

Reply:
Done